tools/config_includer/generator.go

```diff
@@ -91,7 +91,7 @@ func main() {
 
 		newUnresolved := make(map[string]string)
 		for name, fn := range unresolved {
-			if filepath.IsAbs(fn) {
+			if strings.HasPrefix(fn, string(filepath.Separator)) || strings.HasPrefix(fn, "/") {
```

Comment:
I think we only should allow slash based paths in the file... So
```suggestion
			if strings.HasPrefix(fn, "/") {
```
should be good enough.

Reply:
I tried that. `filepath.Join` turns the slashes into backslashes on windows though, so we end up with a backslash from here. https://github.com/influxdata/telegraf/blob/185292dbc3b211605de9ee5e25f4dbbbe3860c1a/tools/config_includer/generator.go#L86

Reply:
Understood. Can you please add this as a comment to the code so people will not "simplify" this as I was tempted to do... :-)

Reply:
Oh saw you fixed it more elegantly... Thanks!